packages/pg-query-stream/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
+    "strict": false,
```

Comment:
it will be nice to turn this to `true` one day, but this is fine for the initial port.

Reply:
I think this would be possible after pg-cursor is converted. I will look at pg-cursor next